Makefile

```diff
@@ -166,25 +140,10 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	  FEED_REVISION=`cd $$FEED_DIR; $(REVISION)`; \
 	  echo "Feed $$FEED: repository from $$FEED_GIT_REPO, git branch \"$$FEED_GIT_BRANCH_ESC\", revision $$FEED_REVISION" >> $$VERSION_FILE; \
 	done
-	# copy different firmwares (like vpn, minimal) including imagebuilder
-	for DIR_ABS in $(IB_BUILD_DIR)/imgbldr/bin/*; do \
-	  TARGET_DIR=$(FW_TARGET_DIR)/$$(basename $$DIR_ABS); \
-	  rm -rf $$TARGET_DIR; \
-	  mv $$DIR_ABS $$TARGET_DIR; \
-	  cp $(FW_TARGET_DIR)/$$VERSION_FILE $$TARGET_DIR/; \
-	  for FILE in $$TARGET_DIR/openwrt*; do \
-	    [ -e "$$FILE" ] || continue; \
-	    NEWNAME="$${FILE/openwrt-/kathleen-}"; \
-	    NEWNAME="$${NEWNAME/ar71xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/mpc85xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/squashfs-/}"; \
-	    mv "$$FILE" "$$NEWNAME"; \
-	  done; \
-	done;
+	./assemble_firmware.sh -p "$(PROFILES)" -i $(IB_FILE) -t $(FW_TARGET_DIR) -u "$(PACKAGES_LIST_DEFAULT)"
```

Comment:
@sarumpaet It would be easy for anybody to understand why we need such strange workarounds, if there would be a good comment within the code. I can not remember, what I did when I written that commit, to much time pasted. But I'm looking up now would be:
- git blame
- see commit fa4aafa
- git show fa4aafa
- oh nice, there is a reference, then let's start webbrowser to see what's the original reason is
- #178 oh great. Just have to read 18 comments to get a overview, what's the original reason and why this commit might fix it.

**Not an easy way.**
Quite time consuming for horrible code, uncommented code.
Commit messages with "See url..." is **NOT** helpful. Please don't create indirections in commit messages, especially bug numbers, bug system, url tend to change when moving from one system to another. Describe what you do and why you do it. Especially when you introduce strange code and add a comment into the strange code too. Yes I am also tend to be lazy from time to time. Please poke me then.

But anyway, I can not find any upstream bug report about this problem? Also the first substitution can be done by configuration.
Removing the platform prefix "mpc85xx-generic-" seems to be a good feature for upstream as `make menuconfig` option.

Please as I stated in the other conversation. Think **long term**. It's important unless we will do the same work again, and again, and again, wasting time in porting patches and workarounds.